packages/collector/package.json

```diff
@@ -15,7 +15,8 @@
   "files": [
     "src",
     "CHANGELOG.md",
-    "esm-loader.mjs"
+    "esm-loader.mjs",
+    "register.mjs"
```

Comment:
Suggestion: Choose a better name for "register.js". Maybe "esm-register.mjs" or "esm-hook.mjs".

The naming depends **heavily** on the decision where we load our tracer.